### PR TITLE
Release 5.2.3

### PR DESCRIPTION
**Pull request by SylvainCorlay**
I am not sure this is the best way to make a 5.x release. Any suggestions? @conda-forge/core

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,9 +21,9 @@ requirements:
   run:
     - python
     - ipython >=4.0.0
-    - ipykernel >=4.5.1
-    - traitlets >=4.3.1
-    - widgetsnbextension >=2.0.0
+    - ipykernel >=4.2.2
+    - traitlets >=4.2.1
+    - widgetsnbextension >=1.2.6, ==1.*
```
Comment (mwcraig) [inline recipe/meta.yaml]: I would express this as `>=1.2.6, <2.0`...but what you have may also work.

Comment (SylvainCorlay) [inline recipe/meta.yaml]: ok, that was from the PEP 440 documentation.

**Comment by SylvainCorlay** [comment]
Closing to open another branch.

File: recipe/meta.yaml (hunk 1 above)
Comment (jasongrout) [inline recipe/meta.yaml]: > ok, that was from the PEP 440 documentation.

IIRC, conda does not understand all of the PEP 440 version numbers (for example, I don't think it understands `~`).

**Pull request closed without merging**